sandbox/namespace_linux.go

```diff
@@ -101,18 +101,6 @@ func removeFromGarbagePaths(path string) {
 	gpmLock.Unlock()
 }
 
-// GC triggers garbage collection of namespace path right away
-// and waits for it.
-func GC() {
```

Comment:
We probably need a `GC` definition in _unsupported to satisfy windows build but we should not be removing the linux definition. That is probably why circle-ci is failing